src/PSF-2.0.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license licenseId="PSF-2.0" name="Python Software Foundation License 2.0">
```

Comment:
In the `license` tag, can you also add another attribute: `listVersionAdded="3.8"`